tests/GovUk.Frontend.AspNetCore.IntegrationTests/StartupFilterTests.cs

```diff
@@ -35,7 +35,7 @@ public async Task HostsCompiledAssets(string fileName)
 
     [Theory]
     [InlineData("images/favicon.ico")]
-    [InlineData("images/govuk-crest.png")]
+    [InlineData("images/govuk-crest.svg")]
```

Comment:
govuk-frontend v5.7.0:

https://github.com/alphagov/govuk-frontend/pull/5376

Note that the CSS changes etc. are all self-contained in the upstream govuk-frontend lib, no need for any additional changes here, beyond the sanity check done as part of this test.


![image](https://github.com/user-attachments/assets/810c1036-9c58-4305-8fae-3d17961076c4)